Doc/library/signal.rst

```diff
@@ -503,3 +503,37 @@ be sent, and the handler raises an exception. ::
 
    signal.alarm(0)          # Disable the alarm
 
+Note on SIGPIPE
+---------------
+
+Do not set set :const:`SIGPIPE`'s disposition to :const:`SIGDFL`
```

Comment:
_Set_ is doubled.

I think SIG_DFL has an underscore, both in Python and in C.